packages/react-examples/src/react/OverflowSet/OverflowSet.Vertical.Example.tsx

```diff
@@ -14,25 +15,28 @@ const onRenderOverflowButtonStyles = {
 
 const onRenderItem = (item: IOverflowSetItemProps): JSX.Element => {
   return (
-    <CommandBarButton
-      role="menuitem"
-      aria-label={item.name}
-      styles={onRenderItemStyles}
-      iconProps={{ iconName: item.icon }}
-      onClick={item.onClick}
-    />
+    <TooltipHost content={item.title} id={`menuitem-${item.key}-tooltip`}>
```

Comment:
Two things:
- I don't think you need the id here (try testing without it)
- To make it look nicer, I think you should add `directionalHint={DirectionalHint.rightCenter}`